src/languages/sqf.js

```diff
@@ -1907,10 +2022,9 @@ export default function(hljs) {
     'setEditorObjectScope',
     'setEffectCondition',
     'setEffectiveCommander',
-    'setEngineRPMRTD',
     'setEngineRpmRTD',
     'setFace',
-    'setFaceAnimation',
+    'setFaceanimation',
```

Comment:
```suggestion
    'setFaceAnimation',
```
https://community.bistudio.com/wiki/setFaceAnimation